sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -83,6 +83,22 @@ object Radar {
 
     }
 
+    /**
+     * Called when a trip update succeeds, fails, or times out.
+     */
+    interface RadarTripCallback {
+
+        /**
+         * Called when a trip update succeeds, fails, or times out. Receives the request status.
+         *
+         * @param[status] RadarStatus The request status.
+         */
+        fun onComplete(
+            status: RadarStatus
```

Comment:
perhaps I lack context here but do we want to return any info in addition to `status`? Maybe the updated `tripOptions` that results from the update?

Reply:
This callback is used with `startTrip(tripOptions, callback)` and `updateTrip(tripOptions, status, callback)`, so the developer will be passing in `tripOptions` already. In a future release we could potentially return `trip`, `user`, or `events` in the callback.